packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -249,6 +251,12 @@ class SemanticsData extends Diagnosticable {
   /// if this node represents a text field.
   final TextSelection textSelection;
 
+  /// The count of children of a scroll node which contribute semantics.
```

Comment:
Total count? Visible count? That's the guidance for the case when the child count is virtually unlimited?